draft-ietf-acme-acme.md

```diff
@@ -1821,7 +1843,7 @@ action the client should take:
   fulfilled, and is awaiting finalization.  Submit a finalization
   request.
 
-* "processing": The certificate is being issued. Send a GET request after the
+* "processing": The certificate is being issued. Send a POST-as-GET request after the
```

Comment:
This request is for the actual certificate, so should be GET or POST-as-GET.

Reply:
My thinking here is to have the description focus on the more secure case (which should be the default for most clients), and let the MAY for GETs stand alone above.